geomstats/geometry/product_manifold.py

```diff
@@ -80,10 +84,11 @@ def belongs(self, point, point_type=None):
             Point.
         point_type : str, {'vector', 'matrix'}
             Representation of point.
+            Optional, default: None.
 
         Returns
         -------
-        belongs : array-like, shape=[..., 1]
+        belongs : array-like, shape=[...,]
             Array of booleans evaluating if the corresponding points
```

Comment:
plural form here --> should be singular?

Reply:
Done.